src/Uno.UI/UI/Xaml/DurationHelper.cs

```diff
@@ -0,0 +1,30 @@
+#pragma warning disable 108 // new keyword hiding
+#pragma warning disable 114 // new keyword hiding
+namespace Windows.UI.Xaml
+{
+	public  partial class DurationHelper 
+	{
+		public static Duration Automatic
+			=> Duration.Automatic;
+
+		public static Duration Forever
+			=> Duration.Forever;
+
+		public static int Compare(Duration duration1, Duration duration2)
+			=> Duration.Compare(duration1, duration2);
+
+		public static Duration FromTimeSpan(global::System.TimeSpan timeSpan)
+			=> new Duration(timeSpan);
+
+		public static bool GetHasTimeSpan(Duration target) => target.HasTimeSpan;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'target' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5475653)